ui/app/components/namespace-picker.js

```diff
@@ -25,10 +25,12 @@ export default class NamespacePicker extends Component {
   @service store;
 
   // Show/hide refresh & manage namespaces buttons
-  @tracked showAfterOptions = false;
+  @tracked hasListPermissions = false;
 
   @tracked selected = {};
   @tracked options = [];
+  @tracked search = '';
+  @tracked namespaceLabel = 'All Namespaces';
```

Comment:
If the title does not change it does not need to be tracked. If there is only one instance of it, hardcode it in the hbs file, otherwise you can assign it here as: namespaceLabel = 'All Namespaces'. Tracked properties are helpful when we expect them to be changed based of of something happening either to arguments passed in or a user action. 
AI overview below:
>In Ember.js, you should use @tracked properties for any property that is mutated and potentially externally visible. This includes properties that are used in templates or computed properties, or that might be accessed or updated by other components or services. Essentially, if a property's change needs to be reflected in the UI or in other parts of your application, it should be tracked.

Reply:
it will change - in my next PR i'm implementing search and it will switch between "All namespaces" and "Matching namespaces" based on whether or not the user has typed anything in the search field.

Reply:
I think the functionality here is going to be added in a follow-on PR because this will updated to be "All namespace" or "Matching namespaces"

Shannon - when the time comes I think this would make sense as a getter that updates dynamically if/when a filter is applied

something like the following (`searchInput` is just a placeholder, whatever you decide or have in mind works)

```js
get namespaceLabel() {

return this.searchInput === '' : "All namespaces" : "Matching namespaces" 
}